dwave/preprocessing/include/dwave/presolve.h

```diff
@@ -344,9 +362,11 @@ class Presolver {
         bool ret = false;
 
         ret |= remove_zero_biases(model_.objective);
-        for (size_t c = 0; c < model_.num_constraints(); ++c) {
-            ret |= remove_zero_biases(model_.constraint_ref(c));
-        }
+        auto constraints = model_.constraints;
+        std::for_each(std::execution::par_unseq, constraints.begin(), constraints.end(),
```

Comment:
I think, theoretically, using [taskflow.for_each](https://taskflow.github.io/taskflow/index.html#QuickStartLeverageStandardParallelAlgorithms) should give better performance, at least in a complex graph.

I find myself wondering if each `technique_` should return a Task instead, or maybe we should have a `task_` set of methods as well, each returning a Task.